Sources/TuistKit/Commands/ShareCommand.swift

```diff
@@ -23,7 +23,7 @@ public struct ShareCommand: AsyncParsableCommand, HasTrackableParameters {
     var path: String?
 
     @Argument(
-        help: "The app names to be looked up in the built products directory or the paths to the app bundles.",
+        help: "The app names to be looked up in the built products directory or the paths to the app bundles or application zip archives.",
```

Comment:
Is the application `.zip` archive a zip containing the `.ipa`?

Reply:
`.ipa` is a special kind of `.zip` but let me update it to reference the `.ipa` since we moved it there.